src/read.lisp

```diff
@@ -400,7 +425,7 @@
               symbol
               (error "The symbol `~S' is not external in the package ~S." name package))))))
 
-(defun read-integer (string)
+#+nil(defun read-integer (string)
```

Comment:
We probably should just remove this instead of commenting it out?

Reply:
Yes off course/
Already